tests/setup_test.py

```diff
@@ -35,37 +49,21 @@ def test_items_tags_setup(item_manager, tag_manager):
 
     Return: list of ids of the added items
     """
-    item_ids = []
+    item_ids = (ITEM_ID_1, ITEM_ID_2, ITEM_ID_3, ITEM_ID_4)
```

Comment:
Since this is not used (?) you can just return the tuple later, instead of introducing a variable that does not provide anything.

Reply:
updated